packages/manager/apps/dbaas-logs/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@ovh-ux/manager-dbaas-logs-app",
+  "version": "0.0.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/apps/dbaas-logs"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "scripts": {
+    "build": "yarn build:eu && yarn build:ca && yarn build:us",
+    "build:ca": "webpack --env.production --env.region='CA' && mv dist dist-CA",
+    "build:eu": "webpack --env.production && mv dist dist-EU",
+    "build:us": "webpack --env.production --env.region='US' && mv dist dist-US",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-dbaas-logs-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-dbaas-logs-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-dbaas-logs-app' --include-dependencies -- npm run dev:watch --if-present"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "^0.4.0",
+    "@ovh-ux/manager-core": "^8.0.0 || ^9.0.0",
+    "@ovh-ux/manager-dbaas-logs": "^0.0.0",
+    "@ovh-ux/ng-ovh-api-wrappers": "^4.0.7",
+    "@ovh-ux/ng-ovh-doc-url": "^1.0.2",
+    "@ovh-ux/ng-ovh-http": "^4.0.4",
+    "@ovh-ux/ng-ovh-request-tagger": "^1.1.0",
+    "@ovh-ux/ng-ovh-sidebar-menu": "^8.5.1",
+    "@ovh-ux/ng-ovh-sso-auth": "^4.2.3",
+    "@ovh-ux/ng-ovh-swimming-poll": "^4.0.1",
+    "@ovh-ux/ng-tail-logs": "^2.0.2",
+    "@uirouter/angularjs": "^1.0.23",
+    "angular": "^1.7.5",
+    "angular-aria": "^1.7.8",
+    "angular-cookies": "^1.7.8",
+    "angular-dynamic-locale": "^0.1.37",
+    "angular-i18n": "^1.7.8",
+    "angular-moment": "^1.0.0",
+    "angular-resource": "^1.7.8",
+    "angular-sanitize": "^1.7.8",
+    "angular-translate": "^2.18.1",
+    "angular-translate-loader-pluggable": "^1.3.1",
+    "angular-ui-bootstrap": "1.3.3",
+    "angular-websocket": "^2.0.0",
+    "bootstrap": "~3.3.0",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
```

Comment:
Maybe we could use latest version

Reply:
We are using some classes from v3 and some from v4. We will have to take this up separately to remove v3 from the entire manager

Reply:
What I meant is
```suggestion
    "bootstrap4": "^4.4.0",
```